pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -589,11 +615,13 @@ func (tc *testContext) listAll(t *testing.T) (objects []metav1.Object) {
 	claims, err := tc.client.ResourceV1alpha2().ResourceClaims("").List(tc.ctx, metav1.ListOptions{})
 	require.NoError(t, err, "list claims")
 	for _, claim := range claims.Items {
+		claim := claim
```

Comment:
Would it be more readable if you use DeepCopy or at least a variable with a different name?

Reply:
no, we explicitly want to shadow so we can't accidentally use the loop var across iterations

Reply:
and there's no need to deepcopy, a var local to the loop is sufficient